src/frontend/src/utils/urls.ts

```diff
@@ -179,6 +179,8 @@ const carsCreate = () => `${cars()}/create`;
 const recruitment = () => `${API_URL}/recruitment`;
 const allMilestones = () => `${recruitment()}/milestones`;
 const milestoneCreate = () => `${recruitment()}/milestone/create`;
+const milestoneDelete = (id: string) => `${recruitment()}/milestone/${id}/delete`;
```

Comment:
move milsetone delete to be the last one in milestones and faqDelete to be the last one in faqs and same for on line 331 and 332, just for organization